src/components/Page.js

```diff
@@ -0,0 +1,16 @@
+import React, { useEffect } from "react"
+import Container from "./Container"
+
+function Page(props) {
+  useEffect(() => {
+    document.title = `${props.title} | Fitigai`
+    window.scrollTo(0, 0)
+  }, [props])
```

Comment:
Interesting way to handle the headers data. But I recommend you to use a library for that.
Look for [React helmet](https://github.com/nfl/react-helmet) for example which is a very popular and famous lib for that needs